nservicebus/messaging/forwarding.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Message forwarding
+summary: Describes how to set up message forwarding
+tags:
+- Forwarding Messages
+redirects:
+---
+
+
+Use this feature to forward successfully processed messages from an endpoint to a specified destination endpoint. If there is a need to collect all successfully processed messages from all endpoints, then use the [Auditing Feature](/nservicebus/operations/auditing.md) instead.
```

Comment:
I'm all ears for suggestions. Since this PR has already been merged, create a new PR?